docs-src/concepts/what-is-the-temporal-web-ui.md

```diff
@@ -31,8 +31,8 @@ For details, see [Namespace-level permissions](/cloud/users-namespace-level-perm
 
 ### Recent Workflows
 
-Recent Workflows lists all Workflow Executions run in the past 24 hours.
-The default number shown is 1,000 Workflow Executions.
+The main Workflows page displays a table of recent Workflow Executions.
+The Web UI will display a maximum of 1,000 Workflow Executions.
```

Comment:
```suggestion
The Web UI displays a maximum of 1,000 Workflow Executions.
```

Most of the time (could be a pun) aim for present tense similar to the previous sentence.